protocol/src/gen_proto.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-# flatc version 1.10.0
+# flatc version 1.11.0
 
 flatc --rust protocol.fbs
```

Comment:
Check the flatc version here to ensure we are using the correct version, like `check-cfbc-version` in Makefile.